src/electron/build_linux_release.action.sh

```diff
@@ -42,25 +42,18 @@ if ((STAGING_PERCENTAGE <= 0)) || ((STAGING_PERCENTAGE > 100)); then
   exit 1
 fi
 
-TAG=$(scripts/get_tag.sh linux)
-if [[ $TAG =~ ^.*-beta$ ]]; then
-  INFO_FILE_CHANNEL="beta"
-else
-  INFO_FILE_CHANNEL="latest"
-fi
-
-npm run action src/electron/package_common linux --buildMode=release
+npm run action src/electron/build_common -- linux --buildMode=release
 
 # Publishing is disabled, updates are pulled from AWS. We use the generic provider instead of the S3
 # provider since the S3 provider uses "virtual-hosted style" URLs (my-bucket.s3.amazonaws.com)
 # which can be blocked by DNS or SNI without taking down other buckets.
-electron-builder \
-  --linux \
+electron-builder --linux \
   --publish never \
   --config src/electron/electron-builder.json \
   --config.extraMetadata.version=$(node scripts/get_version.mjs linux) \
   --config.generateUpdatesFilesForAllChannels=true \
   --config.publish.provider=generic \
   --config.publish.url=https://s3.amazonaws.com/outline-releases/client/linux
 
-echo "stagingPercentage: $STAGING_PERCENTAGE" >> build/dist/$INFO_FILE_CHANNEL-linux.yml
+echo "stagingPercentage: $STAGING_PERCENTAGE" >> build/dist/beta-linux.yml
```

Comment:
I don't really have any context on these staging_percentage things. do we need two separate files anymore?

Part of a larger convo - there's a large amount over overlap now between `build_linux` and `build_linux_release` actions. Main difference being this staging percentage thing and the additional build flags. How would you feel about me merging the two scripts so `build_linux --buildMode=release` effectively calls `build_linux_release`? I could also move these to node as well.

Reply:
One is for the beta channel and one for the stable. But I'm not sure whether we are using them.
For now, outputting them is fine. They will just be in sync.

We do want to use staged rollouts.

Reply:
Of course - I'm going to update it to match the `--buildMode` flagging style as well